easybuild/easyblocks/g/gromacs.py

```diff
@@ -70,6 +70,26 @@ def __init__(self, *args, **kwargs):
         self.lib_subdir = ''
         self.pre_env = ''
 
+        # check whether PLUMED is listed as a dependency
+        self.with_plumed = 'PLUMED' in [dep['name'] for dep in self.cfg['dependencies']]
```

Comment:
I mimiced the code from dl_poly_classic.py which was the only one i could find that used plumed. But I can move it to the prepare step.